tests/retval-atoi-decomp.c

```diff
@@ -0,0 +1,87 @@
+void _init(uint64_t rip)
+{
+	if (*(uint64_t*)0x600ff8 != 0)
+	{
+		__gmon_start__(4195389);
```

Comment:
Down the line I think we can do some special casing for some platforms. E.g. if `--os` is `linux`, then we can do things like omit `__stack_chk_fail`, `__libc_start_main`, `_start`, `register_tm_clones`, `deregister_tm_clones`, etc.